pkg/services/featuremgmt/registry.go

```diff
@@ -110,5 +110,11 @@ var (
 			State:           FeatureStateAlpha,
 			RequiresDevMode: true,
 		},
+		{
+			Name:            "intentapi",
+			Description:     "Enable experimental Intent API",
+			State:           FeatureStateAlpha,
```

Comment:
kinda wish we had more gradation here, but it's a TODO for us anyway to define the messaging around the maturity of this new system, so parking the feature state in alpha is the best thing for now